src/wp-includes/functions.php

```diff
@@ -9090,3 +9090,139 @@ function wp_is_heic_image_mime_type( $mime_type ) {
 
 	return in_array( $mime_type, $heic_mime_types, true );
 }
+
+
+/**
+ * Outputs the tooltip for a specific field.
+ *
+ * @since 6.7.0
+ *
+ * @param string $field_id             The unique identifier for the tooltip container.
+ * @param string $tooltip_text         The text content to be displayed in the tooltip.
+ * @param string $tooltip_button_label Optional. The label for the tooltip button. Default is 'Help'.
+ * @param string $position             Optional. The default position of the tooltip. Default is 'right'.
+ * 									   Accepted values: 'right', 'left', 'up' and 'down'.
+ */
+function add_tooltip( $field_id, $tooltip_text, $tooltip_button_label = 'Help', $position = 'right' ) {
```

Comment:
Let's make the arguments into an array, so that the order isn't important. Somebody should be able to set the position without needing to define the button label.